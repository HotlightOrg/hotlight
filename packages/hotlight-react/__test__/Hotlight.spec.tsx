import React from "react";
import renderer from "react-test-renderer";
import HotlightProvider from "../src/index";

test("render children", () => {
  const component = renderer.create(
    <HotlightProvider>
      <div>Some div</div>
    </HotlightProvider>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
