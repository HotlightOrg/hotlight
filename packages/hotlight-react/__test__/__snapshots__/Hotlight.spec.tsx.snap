// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render children 1`] = `
Array [
  <div>
    Some div
  </div>,
  <hotlight-modal />,
]
`;
